### PR TITLE
Updated User Guide

### PR DESCRIPTION
**Pull request by nicholasjnelson**
This is the first part of the updated User Guide outlined at the 2022 Hack-a-thon. It contains new Getting Started, Physics and Math, and Model Setup sections. Please don't merge until additional pieces are complete.

### CODE REVIEW

File: doc/source/User_Guide/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.. _contribute:
+
+Contributing to Rayleigh
```
Comment (gassmoeller) [inline doc/source/User_Guide/contribute.rst:3]: We do not need to create a new file for this. There is a file Contributing.md in the main directory that is already integrated in doc/source/User_Guide/index.rst. Just link to it here (or remove the contribute.rst completely and instead link Contributing.md). If you want to make changes to our current text, feel free to change Contributing.md directly.

**Review by gassmoeller — COMMENTED** [review]
Just a quick note before you make yourself unnecessary work with the contribute.rst

**Review by feathern — APPROVED** [review]
Merging this now so we can move forward on the revised documentation together.

**Comment by gassmoeller** [comment]
Wait

**Comment by gassmoeller** [comment]
Ah, I guess we can fix my comment in a follow-up.

**Pull request merged**